Project.toml

```diff
@@ -4,6 +4,7 @@ authors = ["Frederik Banning <frederik.banning@ruhr-uni-bochum.de>"]
 version = "0.0.4"
 
 [deps]
+GeometryBasics = "5c1252a2-5f33-56bf-86c9-59e7332b4326"
```

Comment:
I think this dependency is not needed because we can simply do `import Makie.GeometryBasics`.

Reply:
True, I was trying to find it by typing `Makie.Ge<tab>` but that did not expand to anything.  Fixed now.